cmd/kubeadm/app/util/system/docker_validator.go

```diff
@@ -81,6 +84,10 @@ func (d *DockerValidator) unmarshalDockerInfo(b []byte, info *dockerInfo) error
 
 func (d *DockerValidator) validateDockerInfo(spec *DockerSpec, info dockerInfo) ([]error, []error) {
 	// Validate docker version.
+	if info.ServerErrors != nil {
+		return nil, []error{errors.Errorf(`error verifying docker info: "%s"`, strings.Join(info.ServerErrors, `", "`))}
```

Comment:
`error verifying docker info: "%s"` can be changed to "error verifying docker info: %q"
notice, not using `

Reply:
Thanks, totally forgot about `%q`